user/admin.py

```diff
@@ -1,5 +1,27 @@
 from django.contrib import admin
-from django.contrib.auth.admin import UserAdmin
+from django.contrib.auth.admin import UserAdmin as DjangoUserAdmin
+from django.utils.translation import gettext_lazy as _
+
 from .models import User
```

Comment:
use absolute imports instead of relative

Reply:
Revise the code to use absolute imports in the future